src/current/cockroachcloud/serverless-cluster-management.md

```diff
@@ -110,6 +110,13 @@ Use the [Managed-Service Backups]({% link cockroachcloud/use-managed-service-bac
 
 You can also [back up and restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your CockroachDB {{ site.data.products.serverless }} cluster manually. You can take [backups locally]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) to [`userfile`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/use-userfile-storage) or [back up to cloud storage]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}).
 
+## Enable deletion protection
+
+To help prevent a cluster from being deleted by mistake, you can enable _deletion protection_. Before a cluster with deletion protection enabled can be deleted, deletion protection must be disabled. A user with permission to delete a cluster can enable deletion protection on the same cluster.
```

Comment:
```suggestion
To help prevent a cluster from being deleted by mistake, you can enable _deletion protection_. Before you can delete a cluster with deletion protection enabled, you must disable deletion protection. A user with permission to delete a cluster can enable deletion protection on the same cluster.
```

Reply:
Total suggestion. I found that I had to re-read this phrase a few times "... cluster with deletion protection enabled can be deleted...". It may be that I tripped up on the two past participles. (But that might just be me!)

Reply:
If you do change this in any way, there is the other instance of it in cluster management.